web/models.py

```
@@ -2195,47 +2210,85 @@ def __str__(self):
         user_str = self.user.username if self.user else f"Anonymous ({self.anonymous_id})"
         return f"{user_str} - {self.quiz.title}"
 
+    # return percentage from 0% to 100%
     def calculate_score(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `calculate_score`

(ANN201)